libsql-server/src/query_analysis.rs

```diff
@@ -128,9 +128,16 @@ impl StmtKind {
             Cmd::Stmt(Stmt::Attach {
                 expr: Expr::Id(Id(db_name)),
                 ..
-            }) => Some(Self::Attach(
-                NamespaceName::from_string(db_name.to_string()).ok()?,
-            )),
+            }) => {
+                let db_name = db_name
+                    .strip_prefix('"')
+                    .unwrap_or(db_name)
+                    .strip_suffix('"')
+                    .unwrap_or(db_name);
```

Comment:
https://doc.rust-lang.org/std/primitive.str.html#method.trim_matches would that not work?

Reply:
I wanted to remove only at the only at start and end, not inside of the string. They are not expected to occur anywhere else in the normal flow and parser will fail it anyways. What do you say?